js_modules/dagster-ui/packages/ui-core/src/assets/AssetMetadata.tsx

```diff
@@ -9,10 +9,10 @@ import {METADATA_ENTRY_FRAGMENT} from '../metadata/MetadataEntryFragment';
 import {MetadataEntryFragment} from '../metadata/types/MetadataEntryFragment.types';
 
 export const metadataForAssetNode = (
-  assetNode: AssetNodeOpMetadataFragment,
+  assetNode: AssetNodeOpMetadataFragment | null | undefined,
```

Comment:
nit: after talking with dish about it for a bit i was convinced to avoid accepting `undefined` since it is equivalent to omitting the argument entirely. Minor but i think worth considering if only `null` doesn't blow up a bunch of callsites

Reply:
It's not equivalent to omitting the argument entirely. You're required to pass in undefined explicitly.

Reply:
But yeah I'm accepting undefined here because I'm getting this undefined from using `?.` syntax and I believe from apollo itself, so the alternative would be to do ` ?? null` everywhere which seems less appealing since there will be many call sites

Reply:
ah ok ya my misunderstanding about `argument?: Type` vs `argument: Type | undefined`